locales/en/translation.json

```diff
@@ -154,7 +154,7 @@
       "doctype": "The first line of your HTML should always be:\n`<!DOCTYPE html>`",
       "duplicated-id": "You can't use the id \"{{-id}}\" more than once in your HTML",
       "img-src": "`<img>` tags need a `src` attribute, with the URL of the image you want to display.",
-      "href-style": "The `<a>` tag should have an `href` attribute that starts with 'http' or 'https'",
+      "href-style": "The `<a>` tag should have an `href` attribute with a value that starts with `http://`, `https://`, `mailto:`, or `tel:`",
```

Comment:
Hmm sorry for only bringing this up now, but I’m not sure we actually want to mention `mailto:` and `tel:` in the error message—while it’s more technically correct to do so, those are very marginal cases and I worry that mentioning them explicitly will bring more confusion than clarity. Would you be down to keep the behavior in this PR but go back to only mentioning `http://` and `https://` in the error message?